backend/src/main/java/com/htecgroup/skynest/exception/CustomException.java

```diff
@@ -0,0 +1,13 @@
+package com.htecgroup.skynest.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@AllArgsConstructor
+public class CustomException extends RuntimeException {
```

Comment:
instead of CustomException, you can rename it to SkyNestBaseException or something like that.. it is better than Custom :)

Reply:
SkyNestBaseException has a nice ring to it :D